external/storm-jms/src/main/java/org/apache/storm/jms/trident/TridentJmsSpout.java

```diff
@@ -193,7 +185,7 @@ public Fields getOutputFields() {
         private final long rotateTimeMillis;
         private final int maxBatchSize;
         private final String name;
-        private final Logger LOG = LoggerFactory.getLogger(JmsEmitter.class);
+        private final Logger log = LoggerFactory.getLogger(JmsEmitter.class);
```

Comment:
Feel free to make the field static instead.

Reply:
Then the surrounding private class JmsEmitter would need to be static as well. That'd mean touching more than just style. The private class is longer than the surrounding class though, so it could be moved up.

Reply:
Why does the surrounding class need to be static too?

Reply:
Okay, it doesn't compile. Makes sense. I think leaving it like this is fine.